midenc-compile/src/compiler.rs

```diff
@@ -201,6 +201,29 @@ pub struct Compiler {
         )
     )]
     pub link_libraries: Vec<LinkLibrary>,
+    /// Specify path prefixes to try when resolving relative paths from DWARF debug info
+    #[cfg_attr(
+        feature = "std",
+        arg(
+            long = "trim-path-prefix",
+            value_name = "PATH",
+            help_heading = "Compiler"
+        )
+    )]
+    pub trim_path_prefixes: Vec<PathBuf>,
+    /// Print source location information in HIR output
+    ///
+    /// When enabled, HIR output will include #loc() annotations showing the source file,
+    /// line, and column for each operation.
+    #[cfg_attr(
+        feature = "std",
+        arg(
+            long = "print-hir-source-locations",
+            default_value_t = false,
+            help_heading = "Output"
+        )
+    )]
+    pub print_hir_source_locations: bool,
```

Comment:
Let's move this to the `UnstableOptions` struct, which contains other printer-specific compiler options that are sensitive to implementation details of the compiler